content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -31,11 +31,11 @@ If the node where a Pod is running has enough of a resource available, it's poss
 allowed) for a container to use more resource than its `request` for that resource specifies.
 However, a container is not allowed to use more than its resource `limit`.
 
-For example, if you set a `memory` request of 256 MiB for a container, and that container is in
+For example, if you set a memory `request` of 256 MiB for a container, and that container is in
```

Comment:
The literal object set here `memory`, so I think the original is appropriate. The backticks are used to show literal values and the literal value here would actually be `requests`, as in `spec.containers[].resources.requests.memory`.